pkgs/applications/science/medicine/aliza/default.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, stdenv, wrapQtAppsHook, qtbase, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "aliza";
+  version = "1.9.7";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "AlizaMedicalImaging";
+    repo = "AlizaMS";
+    rev = "v${version}";
+    sha256 = "sha256-bambl3NP42KEwh6NPb8jhQlGqaUCSt6QuzUmmBnEAQw=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    pkgs.itk
+    pkgs.cmake
+    qtbase
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE:STRING=Release"
+    "-DALIZA_QT_VERSION:STRING=5"
+    "-DITK_DIR:STRING=${pkgs.itk}"
```

I think it would be good to avoid importing `pkgs`

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, wrapQtAppsHook
, itk
, cmake
, qtbase
}:

stdenv.mkDerivation rec {
  pname = "aliza";
  version = "1.9.7";

  src = fetchFromGitHub {
    owner = "AlizaMedicalImaging";
    repo = "AlizaMS";
    rev = "v${version}";
    sha256 = "sha256-bambl3NP42KEwh6NPb8jhQlGqaUCSt6QuzUmmBnEAQw=";
  };

  nativeBuildInputs = [
    wrapQtAppsHook
  ];

  buildInputs = [
    itk
    cmake
    qtbase
  ];

  cmakeFlags = [
    "-DCMAKE_BUILD_TYPE:STRING=Release"
    "-DALIZA_QT_VERSION:STRING=5"
    "-DITK_DIR:STRING=${itk}"
```